QuestPatcher.Core/Patching/PatchingManager.cs

```diff
@@ -280,6 +280,12 @@ private async Task PatchManifest(ZipArchive apkArchive)
                 });
             }
 
+            if (permissions.Microphone)
+            {
+                Log.Information("Adding microphone permission request . . .");
```

Comment:
Should be no need for this log line imo, the permissions in the `addingPermissions` list are already logged later on.

Reply:
Ah, OK I guess I can remove that